conda_forge_tick/cli.xsh

```diff
@@ -13,7 +13,6 @@ from .auto_tick import main as main_auto_tick
 from .status_report import main as main_status_report
 from .audit import main as main_audit
 from .update_prs import main as main_update_prs
-from .new_update_versions import main as main_populate_versions
```

Comment:
Why was this removed? Don't we still need it in the CLI?

Reply:
We will replace the current one with the new, so this process is redundant now

Reply:
I think the ideia is to remove `update_versions` for the `make_graph` job, and them update the `update version` job at Circle using the `update_versions` instead

Reply:
yup.